src/Voron/Data/Containers/Container.cs

```diff
@@ -402,33 +434,35 @@ public static void Delete(LowLevelTransaction llt, long containerId, long id)
 
             var container = new Container(page);
             var entriesOffsets = container.Offsets;
-            Debug.Assert(entriesOffsets.Length > 0);
+            Debug.Assert(entriesOffsets.Length > 0);            
             var index = (offset - PageHeader.SizeOf) / sizeof(ItemMetadata);
             var metadata = entriesOffsets[index];
             Debug.Assert(metadata.Size != 0);
+            
             int totalSize = 0, count = 0;
             for (var i = 0; i < entriesOffsets.Length; i++)
             {
                 if(entriesOffsets[i].Size ==0)
                     continue;
                 count++;
                 totalSize += entriesOffsets[i].Size;
-            }
-
+            }            
```

Comment:
Minor - formatting: Why some spaces were added to the end of lines here and above?